Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/common/submit-solution/SubmitSolution.tsx

```diff
@@ -0,0 +1,27 @@
+import { AiOutlineSolution } from 'react-icons/ai';
+import { Link } from 'react-router-dom';
+import { IconButton, Tooltip } from '@mui/material';
+
+interface ISubmitSolutionProps {
+    contestId: number;
+    canBeCompeted: boolean;
+}
+
+const SubmitSolution = (props:ISubmitSolutionProps) => {
+    const { contestId, canBeCompeted } = props;
+    return (
+        <Tooltip title="Submit Solution">
+            <IconButton>
+                <Link
+                  to={`/contests/${contestId}/${canBeCompeted
+                      ? 'compete'
+                      : 'practice'}#1`}
```

Comment:
We need to navigate users to different problems in the contest, not just the 1st.

Reply:
If it is referring because of the tests in problem view, then it cannot be done since when user attempts to open problem in a contest in which the user is not registered then it redirects to registration and after that it is navigated to problem # 1